RecoPixelVertexing/PixelLowPtUtilities/python/ClusterShapeHitFilterESProducer_cfi.py

```diff
@@ -11,6 +11,5 @@
 )
 from Configuration.Eras.Modifier_phase2_tracker_cff import phase2_tracker
 phase2_tracker.toModify(ClusterShapeHitFilterESProducer,
-    PixelShapeFile = 'RecoPixelVertexing/PixelLowPtUtilities/data/pixelShape_Phase1Tk.par',
-    doPixelShapeCut = cms.bool(False)
+    PixelShapeFile = 'RecoPixelVertexing/PixelLowPtUtilities/data/pixelShape_Phase2Tk.par',
```

Comment:
is this applicable to T1,T2 geometries?
I thought that the pixelShape_Phase1Tk.par was supposed to be applied to T1,T2 (phase-1 like pixel).
Most of the workflows that we have are still T1. It would be bad if behavior there goes bad with pixelShape_Phase2Tk.